src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -90,7 +100,7 @@ func applyNext() {
 	if httpClient != nil {
 		err := autoupdate.ApplyNext(&autoupdate.Config{
 			CurrentVersion: Version,
-			URL:            serviceURL,
+			URL:            updateServerURL + "/update",
```

Comment:
Because this endpoint is not expected to change regardless of the server URL, I think that keeping this as a constant value is better and future safe (like if we ever want to add another endpoint for whatever reason, we could use `updateServerURL + "/newendpoint" ` or even versioning over the same endpoint `updateServerURL + "/v2.0/update"`).